.gitignore

```diff
@@ -20,6 +20,7 @@
 __pycache__
 /cmd/podman/varlink/ioprojectatomicpodman.go
 /cmd/podman/varlink/iopodman.go
+/pkg/varlink/iopodman.go
```

Comment:
is the old line, line 22 still needed?

Reply:
On a future PR, I'll remove both 21 and 22